.github/release-check-action/check.py

```diff
@@ -44,28 +43,19 @@
         exit_code = 2
         status = "INVALID"
 
-mutation = """mutation AddReleaseComment($input: AddReleaseFileCommentInput!) {
-  addReleaseFileComment(input: $input)
-}"""
-
-mutation_input = {
-    "prNumber": event_data["number"],
-    "status": status,
-    "releaseInfo": release_info,
-}
 
 print(f"Status is {status}")
+print(f"::set-output name=release_status::{status}")
 
-response = httpx.post(
-    API_URL,
-    json={"query": mutation, "variables": {"input": mutation_input}},
-    timeout=120,
-)
-response.raise_for_status()
+if release_info:
+    changelog = release_info["changelog"]
+    changelog = changelog.replace("`", r"\`")
```

Comment:
this was needed for generating the preview, might not be needed anymore, will check soon

Reply:
yes, it is needed: 
![image](https://user-images.githubusercontent.com/667029/128432886-470a7798-4f70-4a7c-aba8-33b0e89fb0d0.png)

Reply:
Does this ^ mean I could write shell code in the changelog and have it run on the action? E.g can I write code that sends the secrets to me via curl?

Reply:
not with the escape, I hope so at least :D I've got another idea to improve this, will test soon